helm/tenant/templates/tenant.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
     name: {{ dig "scheduler" "name" "" . }}
   {{ end }}
 spec:
-  image: {{ dig "image" "repository" "minio/minio" . }}:{{ dig "image" "tag" "RELEASE.2021-10-06T23-36-31Z" . }}
+  image: {{ dig "image" "repository" "minio/minio" . }}:{{ dig "image" "tag" "RELEASE.2021-12-18T04-42-33Z" . }}
```

Comment:
This is the MinIO version that contains the bug affecting the root account

Reply:
Yeah let's wait for another release today @Alevsk